modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,62 +1,68 @@
 @import 'atlas-variables';
+@import 'cadmin-variables';
 
-.lfr-analytics-reports-panel {
-	&.open-admin-panel.sidenav-menu-slider {
-		visibility: visible;
-		width: 330px;
-	}
+html#{$cadmin-selector} {
+	.cadmin {
+		&.lfr-analytics-reports-panel {
+			&.open-admin-panel.sidenav-menu-slider {
+				visibility: visible;
+				width: 320px;
+			}
 
-	.sidebar {
-		border-left: 1px solid $gray-300;
-		box-shadow: none;
-	}
+			.sidebar {
+				border-left: 1px solid $gray-300;
+				box-shadow: none;
+			}
 
-	.sidebar-header {
-		height: 64px;
-		line-height: 32px;
+			.sidebar-header {
+				height: 64px;
+				line-height: 32px;
 
-		.lexicon-icon {
-			font-size: 14px;
-		}
-	}
+				.lexicon-icon {
+					font-size: 14px;
+				}
+			}
 
-	@media only screen and (min-width: 576px) {
-		.sidebar-header {
-			padding-bottom: 12px;
-			padding-top: 12px;
-		}
-	}
+			@media only screen and (min-width: 576px) {
```

Comment:
Should use media query mixin here `@include media-breakpoint-up(sm, $cadmin-grid-breakpoints) {`